api/daemon_test.go

```diff
@@ -33,6 +34,9 @@ func TestStop(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
+	// Sleep to give time for server to close, as /daemon/stop will return success
+	// before Server.Close() is called.
+	time.Sleep(200 * time.Millisecond)
```

Comment:
Seems too complicated for now. I think it makes sense to worry about the server more when we're prepping the API for v1.0. But at the moment we're still refactoring the host and renter.